UM/Qt/Bindings/MeshFileHandlerProxy.py

```diff
@@ -31,9 +32,14 @@ def supportedReadFileTypes(self):
         file_types = []
         all_types = []
 
-        for ext, desc in self._mesh_handler.getSupportedFileTypesRead().items():
-            file_types.append("{0} (*.{1})".format(desc, ext))
-            all_types.append("*.{0}".format(ext))
+        if platform.system() == "Linux":
```

Comment:
It is linux specific because the other platforms do not require it (they care a bit less about case sensitivity), and it looks slightly weird to have all extensions listed twice in the file open dialog.